src/routes/stake/Unstake.svelte

```diff
@@ -357,11 +359,19 @@
 	});
 
 	onMount(() => {
-		const intervalId = setInterval(fetchData, 5000);
-		return () => clearInterval(intervalId);
-	});
+		const intervalIdFetchData = setInterval(fetchData, 5000);
+		const intervalIdQuote = setInterval(async () => {
+			refreshQuoteCounter += 1;
 
-	$: $inputAmount, computeReceiveAmountFastUnstake();
+			if (refreshQuoteCounter >= 10) {
+				await computeReceiveAmountFastUnstake();
+			}
+		}, 100);
+		return () => {
+			clearInterval(intervalIdFetchData);
+			clearInterval(intervalIdQuote);
+		};
```

Comment:
This looks like a poor approach, you just need to refresh the receiveAmountFastUnstake whenever the input is changing.